iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ValidationUtil.java

```diff
@@ -49,18 +49,28 @@ static void checkActivePowerLimits(Validable validable, double minP, double maxP
         }
     }
 
-    static void checkVoltageControl(Validable validable, Boolean voltageRegulatorOn, double voltageSetpoint, double reactivePowerSetpoint) {
+    static void checkTargetDeadband(Validable validable, double targetDeadband) {
+        if (!Double.isNaN(targetDeadband) && targetDeadband < 0) {
+            throw new ValidationException(validable, "Unexpected value for target deadband: " + targetDeadband);
+        }
+    }
+
+    static boolean checkVoltageControl(Validable validable, Boolean voltageRegulatorOn, double voltageSetpoint) {
         if (voltageRegulatorOn == null) {
             throw new ValidationException(validable, "voltage regulator status is not set");
         }
-        if (voltageRegulatorOn) {
+        if (Boolean.TRUE.equals(voltageRegulatorOn)) {
```

Comment:
Why? the previous version was ok

Reply:
Yes, not really sure why I changed it... Sonar considered it as "risky" since it can return a `NullPointerException` but the `null` case is handled above. I will reverse it.